### PR TITLE
Edit Autonity Contract Interface for aut `v0.3.0` and remove commented out `nodejsconsole` commands

### LINKED ISSUE #141: Update Contract Interfaces with support added by aut v0.3.0 Release
### Description

The [`aut v0.3.0` Release](https://github.com/autonity/aut/releases/tag/v0.3.0) has as part of its changes added commands for new Autonity Contract protocol functions.

The docs should be updated to add this support to the [Autonity Contract Interface](https://docs.autonity.org/reference/api/aut/) section of the docs to keep docs `aut` usage in line with the current `aut` Release.

### Rationale

- docs.autonity.org usage is up to date with the usage support provided by the `aut` tool

### Implementation

Review the added commands in the `aut protocol` command group and update to add the new support:

- [ ] Autonity Contract Interface page (https://docs.autonity.org/reference/api/aut/)
- [ ] Autonity Contract Interface, Operator and Protocol only sub-page (https://docs.autonity.org/reference/api/aut/op-prot/)
- [ ] Update

### PR DESCRIPTION
**Pull request by jay-cousins**
### Description

The [`aut v0.3.0` Release](https://github.com/autonity/aut/releases/tag/v0.3.0) has as part of its changes added commands for new Autonity Contract protocol functions.

This PR updates the docs to add this support to the [Autonity Contract Interface](https://docs.autonity.org/reference/api/aut/) section of the docs to keep docs `aut` usage in line with the current `aut` Release.

Changes made:

Aut Protocol function update:

- config: updated response structures
- getBlockPeriod: added `aut` command
- getEpochPeriod: added `aut` command
- getEpochFromBlock: added `aut` command
- getTreasuryAccount: added to docs, `aut` and rpc usage 
- getTreasuryFee - added `aut` and rpc usage examples
- setAccountabilityContract - added `aut` usage
- setACUContract: added `aut` usage
- setStabilizationContract: added `aut` usage
- setSupplyControlContract: added `aut` usage
- setOracleContract: added `aut` usage

Note no additions were made for the `aut protocol` commands to access public variables - `commission-rate-precision` and `last-epoch-block`. Commission rate precision is returned as part of `config` and last epoch block is returned by `get-last-epoch-block`.

Also:

- commented out text removed to keep docs clean.
closes #141

### CODE REVIEW

File: content/docs/reference/api/aut/op-prot/_index.md [hunk 1]
```diff
@@ -442,14 +416,6 @@ aut protocol set-minimum-base-fee [OPTIONS] base-fee
 {{< /tab >}}
 {{< /tabpane >}}
 
-<!--
```
Comment (lorenzo-dev1) [inline content/docs/reference/api/aut/op-prot/_index.md:445]: So we are sure to not use the nodejsconsole anymore?

**Review by lorenzo-dev1 — APPROVED** [review]
LGTM except for this question https://github.com/autonity/docs.autonity.org/pull/142/files#r1457427805

File: content/docs/reference/api/aut/op-prot/_index.md (hunk 1 above)
Comment (jay-cousins) [reply content/docs/reference/api/aut/op-prot/_index.md:445]: Most of the commented out sections got removed during the last docs update - so only these few remain.

It's easy to put them back in.

Comment (jay-cousins) [reply content/docs/reference/api/aut/op-prot/_index.md:445]: I've changed the PR name to make it easy to find should we want to restore `nodejsconsole` commands!

**Pull request merged**